.github/CODEOWNERS

```diff
@@ -30,7 +30,6 @@
 /components/installation-telemetry @gitpod-io/engineering-security-infrastructure-and-delivery
 /components/kots-config-check @gitpod-io/engineering-security-infrastructure-and-delivery
 /install @gitpod-io/engineering-security-infrastructure-and-delivery
-/install/installer @gitpod-io/engineering-security-infrastructure-and-delivery
```

Comment:
Wouldn't removing this line leads to a fallback to the SID team again due to the following line?
```
/install @gitpod-io/engineering-security-infrastructure-and-delivery
```

I think we need this (empty owner) instead, don't we? :thinking: 
```
/install/installer
```

Reply:
You're right, I had missed the above line. Let me update the PR ☺️